examples/CMakeLists.txt

```diff
@@ -5,8 +6,9 @@ include_directories(${CMAKE_SOURCE_DIR} ${CMAKE_SOURCE_DIR}/src)
 foreach(example ${examples})
 	add_executable(example_${example} ${example}.cpp)
 	if (NANODBC_STATIC)
-		target_link_libraries(example_${example} nanodbc ${Boost_LIBRARIES} ${ODBC_LIBRARIES})
```

Comment:
I have missed to review the details of this PR.
This actually drops the fixes in 37c3641a6de17a71442d224a889882cb32937d12 and 631e4496712ea17db2e83aa7a2b8df3c3c36ca08 leading to linker errors on Windows.
Use of `ODBC_LIBRARIES` is required.